incubator/sentry-kubernetes/templates/deployment.yaml

```diff
@@ -28,11 +28,6 @@ spec:
                 key: sentry.dsn
           - name: ENVIRONMENT
             value: {{ .Values.sentry.environment }}
-          - name: RELEASE
-            valueFrom:
-              resourceFieldRef:
-                containerName: {{ .Chart.Name }}
-                resource: image
```

Comment:
Rather than remove this variable please add a if to check if the user passed the `.Values.release`, the default should be empty.